content/showcase.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Showcase"
+description: ""
+summary: ""
+date: 2025-04-08T02:49:45+02:00
+lastmod: 2025-04-08T02:49:45+02:00
+draft: false
+type: "legal"
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+
+## OpenFusion Tools
+
+### OpenFusion Sites
+
+- [Original / Future Public Server Map Site](https://dexlabs.systems/)
+- [Academy Public Server Map Site](https://academy.dexlabs.systems/)
+
+### GitHub
+
+- [OpenFusionMap](https://github.com/OpenFusionProject/OpenFusionMap)
+- [DexLabs Leaderboards](https://github.com/hichemfantar/dex-labs-leaderboards)
+- [Computress Discord Bot](https://github.com/OpenFusionProject/computress-rs)
+- [Scripts](https://github.com/OpenFusionProject/scripts)
+- [FFSpy](https://github.com/OpenFusionProject/FFSpy)
+- [Disunityff](https://github.com/OpenFusionProject/disunityff)
+- [UnityPackFF](https://github.com/dongresource/UnityPackFF)
+- [OpenFusion Server Configurator](https://github.com/Eperty123/OpenFusion-Server-Configurator)
+- [OpenFusion Race Rankings Python Script](https://gist.github.com/CakeLancelot/6e288aecfaf9d640f5b480b48cf82a95)
```

Comment:
Deprecated; replaced by ofapi (https://github.com/OpenFusionProject/ofapi). Same as above, we can keep this one but we should include ofapi higher up on the list.